src/Aquifer.API/Endpoints/Marketing/Unsubscribe/Endpoint.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Aquifer.Data;
+using FastEndpoints;
+
+namespace Aquifer.API.Endpoints.Marketing.Unsubscribe;
+
+public class Endpoint(AquiferDbContext dbContext, ILogger<Endpoint> logger) : Endpoint<Request>
+{
+    public override void Configure()
+    {
+        Get("/marketing/unsubscribe/{UnsubscribeId}");
+        AllowAnonymous();
+    }
+
+    public override async Task HandleAsync(Request req, CancellationToken ct)
+    {
+        try
+        {
+            var subscriberToDisable = dbContext.ContentSubscribers.SingleOrDefault(s => s.UnsubscribeId == req.UnsubscribeId);
```

Comment:
Why aren't we awaiting and using async here?

Reply:
tried await, but ide said that it is not awaitable,  and async is in the method signature.